db_setup.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+)
+
+type Users struct {
+	gorm.Model
+	User_Id      int    `gorm:"AUTO_INCREMENT"`
+	Name         string `gorm:"primary_key"`
+	Batch        string
+	is_master    string
+	EnrollmentNo string
+	Bio          string
+	CreatedAt    time.Time
+	UpdatedAt    time.Time
+}
+
+type Hubs struct {
+	gorm.Model
+	Hub_Id int    `gorm:"AUTO_INCREMENT"`
+	Name   string `gorm:"primary_key"`
+	Bio    string
+	CreatedAt    time.Time
+	UpdatedAt    time.Time
+}
+
+type Events struct {
+	gorm.Model
+	Event_Id    int    `gorm:"AUTO_INCREMENT"`
```

Comment:
As it turns out naming it as ID conflicts with the default column name 'id' already in the postgres db.